### PR TITLE
OBJ moved some unit test under the gov uk branch

### PR DESCRIPTION
**Pull request by mattch1**

### CODE REVIEW

**Review by markpit — APPROVED** [review]

**Review by ICoxCH — APPROVED** [review]

File: src/test/java/uk/gov/companieshouse/api/strikeoffobjections/email/KafkaRestClientUnitTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package email;
+package uk.gov.companieshouse.api.strikeoffobjections.email;
```
Comment (mwestacott) [inline src/test/java/uk/gov/companieshouse/api/strikeoffobjections/email/KafkaRestClientUnitTest.java:1]: You should now be able to remove the import statement below (for KafkaRestClient). This should be flagged by Sonar I'd have thought.

File: src/test/java/uk/gov/companieshouse/api/strikeoffobjections/common/FormatUtilsUnitTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package common;
+package uk.gov.companieshouse.api.strikeoffobjections.common;
```
Comment (mwestacott) [inline src/test/java/uk/gov/companieshouse/api/strikeoffobjections/common/FormatUtilsUnitTest.java:1]: You should now be able to remove the import statement below (for FormatUtils). This should be flagged by Sonar I'd have thought. Same in other classes.

Comment (mwestacott) [reply src/test/java/uk/gov/companieshouse/api/strikeoffobjections/common/FormatUtilsUnitTest.java:1]: Wondering if our Sonar checks are working...

**Review by markpit — APPROVED** [review]

File: src/test/java/uk/gov/companieshouse/api/strikeoffobjections/common/FormatUtilsUnitTest.java (hunk 2 above)
Comment (mattch1) [reply src/test/java/uk/gov/companieshouse/api/strikeoffobjections/common/FormatUtilsUnitTest.java:1]: Removed the imports and the public modifiers as sonar complains about them too. The local ide sonar checks don't seem to appear when you'd expect them it did not seem to complain about this at first. Perhaps it only responds to code changes.

**Pull request merged**